docs/experiments-nfcorpus.md

```diff
@@ -91,6 +90,7 @@ python -m pyserini.encode \
 ```
 
 We're using the [`BAAI/bge-base-en-v1.5`](https://huggingface.co/BAAI/bge-base-en-v1.5) encoder, which can be found on HuggingFace.
+Use `--device cuda` for a faster computation if you have a CUDA-enabled GPU.
```

Comment:
Add a instruction for those who want to run it on GPU. It works on my 3060ti GPU and performance improvement is >7min on CPU to ~20s in GPU.